src/app.ts

```diff
@@ -30,4 +30,8 @@ import { load } from './server-settings';
 
   console.log('Listening on port', port);
 
+  if (!process.env.PUBLIC_URI) {
+    process.env.PUBLIC_URI = 'http://localhost:' + port + '/';
```

Comment:
These new lines should go above `app.listen`, and I  think it should also have a console.log, like `PUBLIC_URI environment variable was not set, defaulting to x`